fun main(): void {
  print_raw(
    try {
      _ = 1 / 0;
      "Fail"
    } catch {
    | DivisionByZeroException _ -> "Pass\n"
    | _ -> "Fail\n"
    }
  )
}
